indico/modules/attachments/__init__.py

```diff
@@ -33,6 +33,25 @@
 
 connect_legacy_signals()
 
+material_types = [
```

Comment:
:+1:

I'd actually add a `get_default_folder_names()` function to attachments/util.py - then we have a single point we can modify once we make it configurable.

In that case also assign the choices of the selectizewidget from the form's `__init__` (after calling super) so you don't access it at import time